crates/holochain_types/src/dht_op.rs

```diff
@@ -133,14 +157,29 @@ impl kitsune_p2p_dht::prelude::OpRegion for DhtOp {
 /// Old alias for DhtOpLite
 pub type DhtOpLight = DhtOpLite;
 
-/// Alias for DhtOpLite
-pub type OpLite = DhtOpLite;
+/// A type for storing in databases that doesn't need the actual
+/// data. Everything is a hash of the type except the signatures.
+#[allow(missing_docs)]
+#[derive(
+    Clone,
+    Debug,
+    PartialEq,
+    Eq,
+    Hash,
+    Serialize,
+    Deserialize,
+    derive_more::Display,
+    derive_more::From,
+)]
+pub enum DhtOpLite {
```

Comment:
What is this enum for?

Reply:
This has been around for a long time, this corresponds to what we actually store in the DhtOp database table, it's a DhtOp without the actual action or entry data included, for data normalization purposes

Reply:
And the SQL migration is used to alter one column to be NULL, though I reverted this in the next PR anyway

Reply:
I know about `DhtOpLite`, but I'm asking why is there a new enum with that name now, when all that is used in the code is `ChainOpLite`?

Reply:
Calling @maackle here, @maackle to the checkout please

Reply:
Oh @jost-s, many uses of ChainOpLite also use `into()` to cast into DhtOpLite, so DhtOpLIte is used quite a lot even if not lexically.

DhtOpLite is just the parallel enum to DhtOp, but "lite". I'm not sure what else to say about it.